Dockerfile

```diff
@@ -10,4 +10,5 @@ VOLUME ["/data"]
 ENV FSS_PATH="/data"
 
 ENTRYPOINT ["/bin/entrypoint"]
+CMD ["--help"]
```

Comment:
We could also set the default command to `run`, but you have to specify `run -c /path/to/config.yml` in most cases anyway. So printing the help text by default seemed the better option.